pkg/controller/controller_utils.go

```diff
@@ -25,8 +25,8 @@ import (
 	"sync/atomic"
 	"time"
 
+	apps "k8s.io/api/apps/v1"
```

Comment:
Use `appsv1` to clearly show we're working with v1 version. It's a pattern used through the entire code base.

Reply:
In this PR, I only wanted to do mechanical translations from `extensions` to `apps`. That makes it easy to see that the transform is the one we intended. If we want to fix the style, I think it should be a separate PR.